### PR TITLE
New TEG sprites

### PR DESCRIPTION
**Pull request by kugamo**
## What Does This PR Do

Adds new sprites for the TEG. 

Also adds red indicator to TEG, akin to pumps.. showing direction setting it has.

## Why It's Good For The Game

I hate soul!

## Images of changes

image

**LATEST STATUS** (note the cleaner pipe connections)
dreamseeker_54ebA3Gn3Q

https://user-images.githubusercontent.com/16618648/175797670-8cd544c6-21ca-4d9d-bbdb-739975a85ae7.mp4

## Changelog
:cl:
imageadd: Added new sprites for TEG.
/:cl:

### CODE REVIEW

**Comment by moxian** [comment]
This is wonderful!

However, may i complain a bit about the lights at the top part of it? The OP has them blinking between green and yellow, which are very similar colors, especially on the white background, which makes them hard to read. I've been looking at it for a minute straight, and i'm still not sure if they are blinking in a specific "direction", or independently, or even whether the lower of the two lights blinks at all. 
May i ask them to have the colors be more distinct (higher contrast I think it's called?)? Or maybe have them blink less rapidly? Or both?..

**Review by hal9000PR — CHANGES_REQUESTED** [review]
instead of doing `overlays += thing` please do `add_overlay(thing)`

File: code/modules/power/generator.dm [hunk 1]
```diff
@@ -36,6 +39,10 @@
 	if(powernet)
 		disconnect_from_network()
 
+/obj/machinery/power/generator/Initialize()
+	..()
```
Comment (SteelSlayer) [inline code/modules/power/generator.dm:43]:
```suggestion
	. = ..()
```

File: code/modules/atmospherics/machinery/components/binary_devices/circulator.dm [hunk 2]
```diff
@@ -108,17 +111,46 @@
...
+		new_pipe_overlay.color = node2.pipe_color
+		add_overlay(new_pipe_overlay)
+	else
+		add_overlay(mutable_appearance(icon, "disconnected"))
 
 	return 1
+
+/obj/machinery/atmospherics/binary/circulator/power_change()
+	. = ..()
+	if((stat & (BROKEN|NOPOWER)))
```
Comment (SteelSlayer) [inline code/modules/atmospherics/machinery/components/binary_devices/circulator.dm:148]:
```suggestion
	if(stat & (BROKEN|NOPOWER))
```

**Review by SteelSlayer — CHANGES_REQUESTED** [review]
Also, can you remove the duplicate  proc for `/obj/machinery/power/generator/power_change()` at the bottom of `code\modules\power\generator.dm`?

**Review by hal9000PR — APPROVED** [review]
LGTM

**Review by SteelSlayer — APPROVED** [review]

**Pull request merged**